recipes/dlt/meta.yml

```diff
@@ -0,0 +1,96 @@
+{% set name = "dlt" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dlt-{{ version }}.tar.gz
+  sha256: 5d6ae7f510084d10c820bc3ca91c1d0708aaf55238d5587ca266583a3f2ea1ae
+
+build:
+  entry_points:
+    - dlt = dlt.cli._dlt:_main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.13
+    - poetry-core >=1.0.8
+    - pip
+  run:
+    - python >=3.8.1,<3.13
+    - requests >=2.26.0
+    - pendulum >=2.1.2
+    - simplejson >=3.17.5
+    - pyyaml >=5.4.1
+    - semver >=2.13.0
+    - hexbytes >=0.2.2
+    - tzdata >=2022.1
+    - tomlkit >=0.11.3
+    - pathvalidate >=2.5.2
+    - sqlalchemy >=1.4.0
+    - typing-extensions >=4.0.0
+    - makefun >=1.15.0
+    - click >=7.1
+    - requirements-parser >=0.5.0
+    - setuptools >=65.6.0
+    - humanize >=4.4.0
+    - astunparse >=1.6.3
+    - gitpython >=3.1.29
+    - pytz >=2022.6
+    - giturlparse >=0.10.0
+    - orjson >=3.6.7,<=3.9.10
+    - tenacity >=8.0.2
+    - jsonpath-ng >=1.5.3
+    - fsspec >=2022.4.0
+    - packaging >=21.1
+    - win-precise-time >=1.4.2
+  run_constrained:
+    - psycopg2-binary >=2.9.1
+    - psycopg2cffi >=2.9.0
+    - grpcio >=1.50.0
+    - google-cloud-bigquery-core >=2.26.0
+    - pyarrow >=12.0.0
+    - duckdb >=0.10.0
```

Comment:
@rudolfix @sh-rp which duckdb version should use?

Reply:
duckdb is an extra for dlt. let's discuss which extras we want to include in the recipe.

Reply:
@rudolfix I removed all optional dependencies and updated version